app/move/controllers/view/view.locals.test.js

```diff
@@ -789,24 +789,56 @@ describe('Move controllers', function () {
           ])
         })
       })
-    })
 
-    context('with feature flag turned on', function () {
-      beforeEach(function () {
-        const locals = proxyquire('./view.locals', {
-          ...pathStubs,
-          '../../../../config': {
-            FEATURE_FLAGS: {
-              YOUTH_RISK_ASSESSMENT: true,
+      context('with feature flag disabled', function () {
```

Comment:
Do we have a corresponding `with feature flag enabled` block anywhere?

Reply:
Yep. By default the feature is (after this change) [enabled here](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/pull/1109/files#diff-0970a4f92dfffb5335068dcec41b22b33b64a586bc06b6adcf68d5e92ded549cR25). So the [other checks](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/pull/1109/files#diff-0970a4f92dfffb5335068dcec41b22b33b64a586bc06b6adcf68d5e92ded549cR357) in this file assert based on this being enabled.

Reply:
I usually prefer being very obvious when working with feature flags, so I'd have a `with enabled` and `with disabled` block, so that when it come to remove it, it is very easy to see what the expected behaviour is.

However, it's not worth holding up this PR for,